spec/factories/projects.rb

```diff
@@ -33,6 +33,14 @@ def yaml_load(file)
     initialize_with { Tmuxinator::Project.new(file, custom_name: "custom") }
   end
 
+  factory :project_with_number_as_name, class: Tmuxinator::Project do
+    transient do
+      file { yaml_load('spec/fixtures/sample_number_as_name.yml') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.